demo/api/src/products/entities/product.entity.ts

```diff
@@ -130,22 +130,23 @@ export class Product extends BaseEntity<Product, "id"> implements DocumentInterf
 
     @Property({ type: "json" })
     @Field(() => [ProductDiscounts])
-    discounts: ProductDiscounts[] = [];
+    discounts?: ProductDiscounts[] = [];
```

Comment:
Why do you need to make this field optional?

Reply:
I don't need to but I decided to do so:
- to have an example where defalutValues is tested
- (more importantly): to make the admin-generator PR work (as it doesn't support this field)